e2e-tests/src/main.rs

```diff
@@ -44,25 +49,58 @@ fn test_finalization(config: Config) -> anyhow::Result<u32> {
     wait_for_finalized_block(connection, 1)
 }
 
-fn test_token_transfer(config: Config) -> anyhow::Result<()> {
+fn test_fee_calculation(config: Config) -> anyhow::Result<()> {
     let Config { node, seeds, .. } = config;
 
-    let accounts: Vec<sr25519::Pair> = accounts(seeds);
-    let from: sr25519::Pair = accounts.get(0).expect("No accounts passed").to_owned();
-    let to = AccountId::from(
-        accounts
-            .get(1)
-            .expect("Pass at least two accounts")
-            .public(),
+    let (from, to) = get_first_two_accounts(&accounts(seeds));
+    let connection = create_connection(node).set_signer(from.clone());
+    let from = AccountId::from(from.public());
+    let to = AccountId::from(to.public());
+
+    let balance_before = balance_of(&from, &connection);
+    info!("[+] Account {} balance before tx: {}", to, balance_before);
+
+    let transfer_value = 1000u128;
+    let tx = transfer(&to, transfer_value, &connection);
+
+    let balance_after = balance_of(&from, &connection);
+    info!("[+] Account {} balance after tx: {}", to, balance_after);
+
+    let FeeInfo {
+        base_fee,
+        len_fee,
+        unadjusted_weight,
+        adjusted_weight,
+        tip,
+    } = get_tx_fee_info(&connection, &tx);
```

Comment:
Maybe the FeeInfo should have just 3 fields:
* `fee`
* `unadjusted_weight`
* `adjusted_weight`

Reply:
thank you, done